angrmanagement/ui/views/jobs.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PySide6.QtCore import QSize
+from PySide6.QtWidgets import QVBoxLayout, QWidget
+
+from angrmanagement.data.instance import Instance
+from angrmanagement.ui.widgets.qjobs import qjobs
+
+from .view import InstanceView
+
+if TYPE_CHECKING:
+    from angrmanagement.data.instance import Instance
+    from angrmanagement.ui.workspace import Workspace
+
+
+class jobsView(InstanceView):
```

Comment:
Classes should follow PascalCase. First letter of each word is uppercase. Also the file should follow the same pattern as the other views, so jobs_view.py instead of jobs.py